tests/migration/paused.go

```diff
@@ -0,0 +1,190 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package migration
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"kubevirt.io/kubevirt/tests/libmigration"
+
+	migrationsv1 "kubevirt.io/api/migrations/v1alpha1"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+
+	kvpointer "kubevirt.io/kubevirt/pkg/pointer"
+
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+	"kubevirt.io/kubevirt/tests/testsuite"
+
+	"kubevirt.io/kubevirt/tests/framework/checks"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"kubevirt.io/kubevirt/tests/libvmifact"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libnet"
+)
+
+var _ = SIGMigrationDescribe("Live Migrate A Paused VMI", func() {
+	var (
+		virtClient kubecli.KubevirtClient
+		err        error
+	)
+
+	BeforeEach(func() {
+		checks.SkipIfMigrationIsNotPossible()
+		virtClient = kubevirt.Client()
+	})
+
+	waitUntilMigrationMode := func(vmi *v1.VirtualMachineInstance, expectedMode v1.MigrationMode, timeout int) *v1.VirtualMachineInstance {
+		By("Waiting until migration status")
+		EventuallyWithOffset(2, func() v1.MigrationMode {
```

Comment:
Shouldn't the offset be 1?
```suggestion
		EventuallyWithOffset(1, func() v1.MigrationMode {
```

Reply:
hm, I guess. :)